docker-compose.yml

```diff
@@ -1,15 +1,16 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:latest
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_4_externalize_configs/assets/87720583/3b95c4e2-984b-428f-9598-5a88fb73382f)
No need to change the version to the `latest`. Read in [doc](https://semver.org/) what semantic versioning means
```suggestion
    image: todoapp:2.0.1
```